src/Serialization/serializers.jl

```diff
@@ -31,6 +31,11 @@ mutable struct SerializerState
   refs::Vector{UUID}
   io::IO
   key::Union{Symbol, Nothing}
+  type_attr_map::Dict{Type, Vector{Symbol}}
+end
+
+function attrs_list(s::SerializerState, T::Type)
+  return get(s.type_attr_map, T, nothing)
```

Comment:
```suggestion
  return get(s.type_attr_map, T, Dict{Type, Vector{Symbol}}())
```
from a usability perspective (I am currently trying to make this PR work together with https://github.com/oscar-system/Oscar.jl/pull/3980) I find this desireable, because I don't have to take care of any possibility of nothings myself (as e.g. in https://github.com/oscar-system/Oscar.jl/pull/4020/files#diff-45089e89c18427deb0ceef25fae6c753eb01fd978a80d507ca509e3caff7bf37R11)